tests/test_bench.py

```
@@ -0,0 +1,23 @@
+import common
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)